lib/FSi/DoctrineExtensions/Uploadable/FileHandler/SplFileInfoHandler.php

```diff
@@ -50,7 +63,13 @@ public function getName($file)
             throw $this->generateNotSupportedException($file);
         }
 
-        return basename($file->getRealpath());
+        $filename = basename($file->getRealpath());
+
+        if (!empty($filename)) {
```

Comment:
I feel like this should only be done for classes that are expected to not have file names, otherwise it means something went wrong.